post.hbs

```diff
@@ -46,27 +46,81 @@ into the {body} of the default.hbs template --}}
             {{/if}}
 
             <footer class="post-full-footer">
-                {{!-- Everything inside the #author tags pulls data from the author --}}
-                {{#author}}
 
-                <section class="author-card">
-                    {{#if profile_image}}
-                        <img class="author-profile-image" src="{{profile_image}}" alt="{{name}}" />
-                    {{/if}}
-                    <section class="author-card-content">
-                        <h4 class="author-card-name"><a href="{{url}}">{{name}}</a></h4>
-                        {{#if bio}}
-                            <p>{{bio}}</p>
+                {{!-- If there are multiple authors for the post, display all of their avatars in a stack --}}
+                {{#has author="count:>1"}}
+
+                    <section class="authors">
+
+                        <div class="names">
```

Comment:
naming is a little confusing here, would expect these to be

.post-full-authors
.post-full-authors-content